doc/script_commands.txt

```diff
@@ -3176,23 +3176,43 @@ Example:
 ---------------------------------------
 
 *getiteminfo(<item ID>, <type>)
+*setiteminfo(<item ID>, <type>, <value>)
 
 This function will look up the item with the specified ID number in the
 database and return the info set by TYPE argument.
 It will return -1 if there is no such item.
 
 Valid types are:
-	0 - Buy Price; 1 - Sell Price; 2 - Item Type;
-	3 - maxchance (Max drop chance of this item e.g. 1 = 0.01% , etc..
+
+	ITEMINFO_BUYPRICE   - Buy Price
+	ITEMINFO_SELLPRICE  - Sell Price
+	ITEMINFO_TYPE       - Item Type
+	ITEMINFO_MAXCHANCE  - Max drop chance of this item e.g. 1 = 0.01% , etc..
 		if = 0, then monsters don't drop it at all (rare or a quest item)
 		if = 10000, then this item is sold in NPC shops only
-	4 - sex; 5 - equip; 6 - weight; 7 - atk; 8 - def; 9 - range;
-	10 - slot; 11 - subtype; 12 - elv; 13 - wlv; 14 - view id
-
-	If RENEWAL is defined, 15 - matk
+	ITEMINFO_SEX        - Sex
+	ITEMINFO_LOC        - Equip location
+	ITEMINFO_WEIGHT     - Weight (note: 1/10 of unit)
+	ITEMINFO_ATK        - Attack
+	ITEMINFO_DEF        - Defense
+	ITEMINFO_RANGE      - Range
+	ITEMINFO_SLOTS      - Slots
+	ITEMINFO_SUBTYPE    - Item subtype
+	ITEMINFO_ELV        - Equip min. level
```

Comment:
What about equip max level? Maybe we should bring that under this command too.

Reply:
I agree. I opened a separate ticket, #1906, so that we can merge this PR (since it contains fixes for a regression that already landed in the stable branch)